jsonnet/components/prometheus.libsonnet

```diff
@@ -565,6 +565,24 @@ function(params)
           },
           {
             name: 'prometheus',
+            volumeMounts+: [
+              {
+                name: $.trustedCaBundle.metadata.name,
+                mountPath: '/etc/pki/ca-trust/extracted/pem/',
+              },
+            ],
+          },
+        ],
+        volumes+: [
+          {
+            name: $.trustedCaBundle.metadata.name,
+            configMap: {
+              name: $.trustedCaBundle.metadata.name,
+              items: [{
+                key: 'ca-bundle.crt',
+                path: 'tls-ca-bundle.pem',
```

Comment:
(suggestion) It might be worth putting [`src/crypto/x509/root_linux.go#L15`](https://cs.opensource.google/go/go/+/master:src/crypto/x509/root_linux.go;l=15?q=%2Fpki&ss=go%2Fgo) in a comment here, but nothing that would block this PR.

Reply:
I'd rather do it once for all components if we feel that it's important.